backend/capellacollab/sessions/files/routes.py

```diff
@@ -63,11 +63,13 @@ def upload_files(
 
         for file in files:
             file.file.seek(0)
-            file.filename = file.filename.replace(" ", "_")
-            tar.addfile(
-                tar.gettarinfo(arcname=file.filename, fileobj=file.file),
-                fileobj=file.file,
-            )
+
+            if file.filename:
```

Comment:
If `file.filename` _should_ me set, you can also add an assert, instead of an if statement.

E.g.

`assert file.filename`

Reply:
Perfect, I prefer the `assert` you suggested, reducing the nesting here.